translations/base-fr.yaml

```diff
@@ -728,75 +729,75 @@ settings:
+                Raccourcit les panneaux d’information sur les bâtiments en n’affichant que les ratios. Si désactivé, montre une description et une image.
 
         disableCutDeleteWarnings:
-            title: Désactive les avertissement pour Couper/Effacer
+            title: Désactive les avertissements pour Couper / Effacer
             description: >-
-                Désactive la boîte de dialogue qui s'affiche lorsque vous vous apprêtez à couper/effacer plus de 100 entités.
+                Désactive la boîte de dialogue qui s’affiche lorsque vous vous apprêtez à couper / effacer plus de 100 entités.
 
         enableColorBlindHelper:
-            title: Mode Daltonien
+            title: Mode daltonien
             description: Active divers outils qui permettent de jouer à ce jeu si vous êtes daltonien.
 
         rotationByBuilding:
             title: Rotation par catégorie de bâtiment
             description: >-
                 Chaque catégorie de bâtiment enregistre le sens de rotation que vous lui avez assigné la dernière fois, de manière individuelle.
-                Cela sera sans doute plus confortable si vous alternez fréquemment entre le placement de différents types de bâtiments.
+                Cela sera sans doute plus agréable si vous alternez fréquemment entre le placement de différents types de bâtiments.
 
         lowQualityMapResources:
             title: Ressources de la carte de plus basse qualité
             description: >-
-                Simplifie le rendu des ressources sur la carte lorsqu'elle est zoomée opur améliorer les performances.
-                C'est encore plus clean, n'oubliez pas d'essayer !
+                Simplifie le rendu des ressources sur la carte lorsqu’elle est zoomée pour améliorer les performances.
+                Ça donne un rendu encore plus propre, alors essayez-le !
 
         disableTileGrid:
-            title: Desactiver la grille de placement
+            title: Désactiver la grille de placement
             description: >-
-                Desactiver la grille de placement peut aider les performances. Ça rend aussi le jeu encore plus uni!
+                Désactiver la grille de placement peut améliorer les performances. Ça rend aussi l’apparence plus unie !
 
         clearCursorOnDeleteWhilePlacing:
-            title: Effacer le curseur avec clic droit
+            title: Vider le curseur avec le clic droit
             description: >-
-                Activé par défaut, efface le curseur lorsque vous faites un clic droit en ayant un bâtiment selectioné pour la constructio. Si desactivé, vous pouvez detruire les bâtiments en faisant un clic droit tout en placant un bâtiment.
+                Activé par défaut, remet à zéro le curseur lorsque vous faites un clic droit en ayant un bâtiment selectionné pour la construction. Si désactivé, vous pouvez détruire les bâtiments avec un clic droit puis continuer de placer un bâtiment.
```

Comment:
Not very happy with this, @Naheulf any better ideas?

Reply:
I think can be explicit to the current behavior but is pretty long:
> Activé par défaut. Désélectionne le bâtiment choisi pour la construction lorsque vous faites un clic droit sur un bâtiment existant. Si désactivé, vous pouvez détruire des bâtiments avec un clic droit puis continuer de placer le bâtiment sélectionné.